eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -43,18 +45,10 @@ steps:
   - ${{ if ne(parameters.skipTests, 'true') }}:
     - script: ${{ parameters.dotnetScript }} dotnet-coverage collect
               --settings $(Build.SourcesDirectory)/eng/CodeCoverage.config
-              --output ${{ parameters.repoTestResultsPath }}/NonHelixTests.cobertura.xml
+              --output ${{ parameters.repoTestResultsPath }}/NonHelixCodeCoverage/NonHelix.cobertura.xml
```

Comment:
Should repoTestResultsPath here also change to repoArtifactsPath?

Reply:
Technically we could, we won't gain anything.